formal/icache/run.sby.j2

```diff
@@ -17,11 +17,14 @@ cv: depth 32
 smtbmc boolector
 
 [script]
-read -sv @INPUT@
+{{"-sv"|gen_reads}}
 
 # Our formal properties are currently just about control logic, which
 # isn't affected by the exact behaviour of the memories in the design.
 # Blackbox them.
 blackbox $abstract\prim_generic_ram_1p
 
 prep -top ibex_icache
```

Comment:
I'd probably use the "toplevel" template variable here as well to ensure changes to the toplevel in the fusesoc .core file get reflected here as well. But fine either way, it's rather hypothetical at this point.

Reply:
I wasn't sure whether the future-proofing was worth the indirection. I'm still on the fence a bit, but let's show off all the features for now :-)